### PR TITLE
Create a page with a card component and working prev/next buttons

### PR DESCRIPTION
**Pull request by izabelladobielewska**

### CODE REVIEW

File: components/Header/index.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export default function Header() {
+  return (
+    <header className="header">
+      <h1 className="header__title">Endstagram</h1>
```
Comment (ninaassmann) [inline components/Header/index.js]: better use only styled components, don't mix them up with classes and css files

**Review by ninaassmann — CHANGES_REQUESTED** [review]
better only styled components

File: components/Header/index.js (hunk 1 above)
Comment (izabelladobielewska) [reply components/Header/index.js]: Hi Nina, 
Thank you for your review! 😊 Damn, you're of course right, gonna change it! (I'm still a little stuck with the old methods of styling... 🤦‍♀️)

File: components/Header/header.css [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.app__card {
```
Comment (CJAkkuec) [inline components/Header/header.css]: I don't think you're using this anywhere, and you probably also shouldn't mix vanilla CSS with styled-components. 🤔

**Review by CJAkkuec — CHANGES_REQUESTED** [review]
Nice job, just one small note. :)

File: components/Header/header.css (hunk 2 above)
Comment (izabelladobielewska) [reply components/Header/header.css]: Thank you CJ! will be removed 🫡

**Review by CJAkkuec — APPROVED** [review]
Ship it 🚀

**Pull request merged**